src/cli/backup/sharepoint.go

```diff
@@ -334,6 +328,21 @@ func sharePointBackupCreateSelectors(
 		return sel, nil
 	}
 
+	return addCategories(sel, cats), nil
+}
+
+func handleWildCard(categories []string) *selectors.SharePointBackup {
+	sel := selectors.NewSharePointBackup(selectors.Any())
+	if len(categories) == 0 {
+		sel.Include(sel.AllData())
+	} else {
+		sel = addCategories(sel, categories)
+	}
```

Comment:
The len check should either get moved into `addCategories`, or the addCategories behavior should fold into `handleWildCard`.

Reply:
Resolved within [3b162ac](https://github.com/alcionai/corso/pull/2580/commits/3b162ac9588ee1f1ece006323b5490b5f29bf284)